spec/integration/data_stream/manifest.spec.yml

```diff
@@ -176,8 +176,9 @@ spec:
       examples:
       - metrics
     hidden:
-      description: Specifies if a data stream is hidden
+      description: Specifies if a data stream is hidden, resulting in dot prefixed system indices
```

Comment:
Any other suggestion for this description ?

For context, the description of the other `hidden` field under `elasticsearch` key is:
```yaml
  properties:                                  
    hidden:                                    
      description: Makes the data stream hidden
      type: boolean                            
```

Reply:
Sounds good, maybe add a reference to the other setting for hidden data streams.

Reply:
Updated description of the top level hidden field